tests/migration.go

```diff
@@ -172,3 +173,29 @@ func RunStressTest(vmi *v1.VirtualMachineInstance, vmsize, stressTimeoutSeconds
 		time.Sleep(15 * time.Second)
 	}
 }
+
+func WaitForMigrationRunning(virtClient kubecli.KubevirtClient, vmim *v1.VirtualMachineInstanceMigration, vmi *v1.VirtualMachineInstance, timeout time.Duration) error {
+	return wait.PollImmediate(1*time.Second, timeout, func() (bool, error) {
```

Comment:
I think you need to wrap the error (if it occurs) with the context of what failed (see next comment).